### PR TITLE
Bug - unifier added duplicate doc links and titles. 

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39922

## Description
Contributions and Docs link was re-added if they were already exsist.
## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by dantavori — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/unify/tests/yml_unifier_test.py [hunk 1]
```diff
@@ -235,6 +235,28 @@ def test_get_integration_doc_link_negative(tmp_path):
     assert integration_doc_link == ''
 
 
+def test_get_integration_doc_link_exist_in_readme(tmp_path):
+    """
+    Given:
+        - integration which have README in the integration dir, with "View Integration Documentation" doc link
```
Comment (dantavori) [inline demisto_sdk/commands/unify/tests/yml_unifier_test.py:241]:
```suggestion
        - integration which have a detailed description with "View Integration Documentation" doc link
```

File: demisto_sdk/commands/unify/yml_unifier.py [hunk 2]
```diff
@@ -465,10 +466,11 @@ def add_contributors_support(self, unified_yml: Dict, contributor_type: str,
 
         return unified_yml
 
-    def get_integration_doc_link(self, unified_yml: Dict) -> str:
+    def get_integration_doc_link(self, unified_yml: Dict, detailed_description='') -> str:
```
Comment (dantavori) [inline demisto_sdk/commands/unify/yml_unifier.py:469]: tip: with your current implementation, you added non-trivial logic (and in particular a non-trivial argument) to the function. the best practice would be to keep the logic here as is, and when you **call** this function, check before if there is already a doc link in the current detailed description, and if not - call the function.

**Review by dantavori — CHANGES_REQUESTED** [review]
almost there 💪🏽

**Review by dantavori — APPROVED** [review]
looks great

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 * Fixed an issue where yml files with '!reference' failed to load properly.
+* Fixed an issue when `View Integration Documentation` button was added twice during the download and re-upload.
+* Fixed an issue when `(Partner Contridution)` was added twice to the display name during the download and re-upload.
```
Comment (tomneeman151293) [inline CHANGELOG.md:4]:
```suggestion
* Fixed an issue when `(Partner Contribution)` was added twice to the display name during the download and re-upload.
```

**Pull request merged**